test/MorphoTokenOptimism.t.sol

```diff
@@ -0,0 +1,127 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.13;
+
+import {Test, console} from "lib/forge-std/src/Test.sol";
+import {MorphoTokenOptimism} from "../src/MorphoTokenOptimism.sol";
+import {ERC1967Proxy} from
+    "lib/openzeppelin-contracts-upgradeable/lib/openzeppelin-contracts/contracts/proxy/ERC1967/ERC1967Proxy.sol";
+import {UUPSUpgradeable} from "lib/openzeppelin-contracts-upgradeable/contracts/proxy/utils/UUPSUpgradeable.sol";
+
+contract MorphoTokenOptimismTest is Test {
+    address internal constant MORPHO_DAO = 0xcBa28b38103307Ec8dA98377ffF9816C164f9AFa;
+    address internal REMOTE_TOKEN;
+    address internal BRIDGE;
+
+    MorphoTokenOptimism public tokenImplem;
+    MorphoTokenOptimism public morphoOptimism;
+    ERC1967Proxy public tokenProxy;
+
+    uint256 internal constant MIN_TEST_AMOUNT = 100;
+    uint256 internal constant MAX_TEST_AMOUNT = 1e28;
+
+    function setUp() public virtual {
+        REMOTE_TOKEN = makeAddr("RemoteToken");
+        BRIDGE = makeAddr("Bridge");
+
+        // DEPLOYMENTS
+        tokenImplem = new MorphoTokenOptimism();
+        tokenProxy = new ERC1967Proxy(address(tokenImplem), hex"");
+
+        morphoOptimism = MorphoTokenOptimism(payable(address(tokenProxy)));
+        morphoOptimism.initialize(MORPHO_DAO, REMOTE_TOKEN, BRIDGE);
+    }
+
+    function testInitializeZeroAddress(address randomAddress) public {
+        vm.assume(randomAddress != address(0));
+
+        address proxy = address(new ERC1967Proxy(address(tokenImplem), hex""));
+
+        vm.expectRevert();
```

Comment:
We could specify the `expectRevert` in this file, to have the matching error

Reply:
As the `require` takes a custom error (and not a string) as imput I couldn't manage to match the error in the `expectRevert`

I gave the error's selector to `expectRevert` but it didn't work

Reply:
It seems to work for me when I do `        vm.expectRevert(OptimismMintableERC20Upgradeable.ZeroAddress.selector);`

Reply:
Oh cool
I'll make the change in another PR then (for all the tests)